src/Symfony/Component/DependencyInjection/Argument/ServiceLocator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class ServiceLocator
```

Comment:
> you're missing eg exceptions - the wording about them in PSR-11 is a requirement here also

What exactly do we miss? You shouldn't access array field without checking whether it exists. So you won't face the `NotFoundException` (the only checked exception in PSR-11, btw) or your code is just broken.